driver.go

```diff
@@ -604,6 +605,83 @@ type ocRows struct {
 	options TraceOptions
 }
 
+// HasNextResultSet calls the implements the driver.RowsNextResultSet for ocRows.
+// It returns the the underlying result of HasNextResultSet from the ocRows.parent
+// if the parent implements driver.RowsNextResultSet.
+func (r ocRows) HasNextResultSet() bool {
+	if v, ok := r.parent.(driver.RowsNextResultSet); ok {
+		return v.HasNextResultSet()
+	}
+
+	return false
+}
+
+// NextResultsSet calls the implements the driver.RowsNextResultSet for ocRows.
+// It returns the the underlying result of NextResultSet from the ocRows.parent
+// if the parent implements driver.RowsNextResultSet.
+func (r ocRows) NextResultSet() error {
+	if v, ok := r.parent.(driver.RowsNextResultSet); ok {
+		return v.NextResultSet()
+	}
+
+	return nil
+}
+
+// ColumnTypePrecisionScale calls the implements the driver.RowsColumnTypePrecisionScale for ocRows.
```

Comment:
Please move around these method implementations to follow the same order as their interface definitions in the driver package.